Clean up `Game mode/osu!` (#9401)
ppy/osu-wiki: wiki/Game_mode/osu!/en.md

> The first game mode and namesake which was based from the ouendan series.

Reviewer: ```suggestion
The first game mode, with gameplay [based on the ouendan series](#inspiration).
```

![Hit circles](/wiki/shared/osu_hitcircles.jpg "osu! hit circles")

> Coloured circles with numbers on top of them, called hit circles, will appear on the playfield when playing. A thin, similarly-coloured [approach circle](/wiki/Gameplay/Hit_object/Approach_circle) on the outside of the hit circle will shrink over time. Tap on the hit circle at the exact point when the approach circle touches its white border. Note that you need to tap these hit circles in the order indicated by the number.

Reviewer: ```suggestion
Coloured circles with numbers on top of them, called hit circles, will appear on the playfield when playing. A thin, similarly coloured [approach circle](/wiki/Gameplay/Hit_object/Approach_circle) on the outside of the hit circle will shrink over time. Tap on the hit circles at the exact point when the approach circle touches their white borders, in the order indicated by the numbers.
```

> A score burst will appear after tapping the hit circle, showing the score and health changes depending on timing of the tap to the beat. The hit circle will disappear after the score burst.

Reviewer: ```suggestion
After hitting a hit circle, a number appears indicating the [judgement](/wiki/Gameplay/Judgement/osu!) received for how accurate the timing of the hit was.
```

#### Sliders

![Sliders](/wiki/shared/osu_slider.jpg "osu! sliders")

> First, tap on the circle at the beginning of the slider (also known as the [sliderhead](/wiki/Gameplay/Hit_object/Slider/Sliderhead)) at the right moment. Once you tap on it, a ball will begin to move across the path. The orange outer circle, called the follow circle, will appear when holding onto the slider's ball, but will disappear when the cursor is outside the circle or the button is released. Hold the mouse/keyboard button (or keep the pen onto the tablet) and follow the ball within the follow circle as it moves.

Reviewer: i suppose this hasn't been propagated throughout the wiki but "slider head" is preferred instead of "sliderhead"

```suggestion
First, tap on the circle at the beginning of the slider, called the [slider head](/wiki/Gameplay/Hit_object/Slider/Sliderhead), at the right moment. When tapped, a ball will begin to move across the path. The orange outer circle, called the follow circle, will appear when holding onto the slider's ball, but will disappear when the cursor is outside the circle or the button is released. Hold the mouse/keyboard button (or keep the pen on the tablet) and follow the ball within the follow circle as it moves.
```

Sometimes, as seen in the screenshot above, the ball may reverse its direction and the player must follow the ball back to the start of the path or vice versa. The visual cue is a reverse arrow at the ending/starting circle of the path.

#### Spinners

![Spinner](/wiki/shared/osu_spinner.jpg "osu! spinners")

Hold on the mouse/keyboard-button (or keep the pen onto the tablet). From there, use the mouse (or pen) and spin the spinner in a circular motion (in either direction) until the spinner circle grows outwards completely. A *Clear* notice will appear to indicate that the spinner was completed. If the spinner was cleared early, you can continue spinning to collect score bonus and gain some health back.

> The outer white circle shows how much time left to complete the spinner. This circle will turn red to notify that time is almost out. Older skins, using *skin version 1.0*, will have a meter/gauge to indicate how close to clearing the spinner.

Reviewer: ```suggestion
The outer white circle shows how much time is left to complete the spinner. This circle will turn red to notify that time is almost out. Older skins, using [skin version](/wiki/Skinning/skin.ini#versions) 1.0, will have a meter/gauge to indicate how close the spinner is to being completed.
```

> The small box below the spinner shows how many spins you can make per minute, if you spin the spinner at current speed for one minute. If the mouse/keyboard button was released (or the pen did not touch the tablet) early during the spin, the spinner will stop reading the spin.

Reviewer: bit redundant i think

```suggestion
The small box below the spinner shows the current spin speed, measured in spins per minute.
```

## Trivia

> ### What is "Ouendan"?
> 
> *[Osu! Tatakae! Ouendan](https://en.wikipedia.org/wiki/Osu!_Tatakae!_Ouendan)* is a rhythm game for the *Nintendo DS*. The gameplay consists of only three elements: tapping circles on the touchscreen, dragging a ball across a fixed path and rotating a spinner very fast. All these elements are timed to (covers of) popular Japanese songs. It looks like this on the *DS*:

Reviewer: ```suggestion
### Inspiration

The gameplay of osu! is based on *[Osu! Tatakae! Ouendan](https://en.wikipedia.org/wiki/Osu!_Tatakae!_Ouendan)*, a rhythm game for the Nintendo DS. Like osu!, the gameplay consists of only three elements: tapping circles on the touchscreen, dragging a ball across a fixed path, and rotating a spinner very fast. All these elements are timed to covers of popular Japanese songs. It looks like this on the DS:
```

![Gameplay example of Osu! Tatakae! Ouendan in Nintendo DS](/wiki/shared/Ouendan.jpg "Gameplay example of Osu! Tatakae! Ouendan in Nintendo DS")

> The gameplay circles can be seen on the bottom screen, and the story on the top screen. Each stage is basically a self-contained story about a person in trouble. That is where the *Ouendan* (Cheer squad) comes in. Through the magical power of male cheerleading, the player have to help the people in trouble.

Reviewer: ```suggestion
The gameplay circles can be seen on the bottom screen, and the story on the top screen. Each stage is basically a self-contained story about a person in trouble. That is where the *Ouendan* (cheer squad) comes in. Through the magical power of male cheerleading, the player has to help the people in trouble.
```
